data/data-pipeline/data_pipeline/etl/score/etl_score.py

```diff
@@ -323,7 +38,27 @@ def extract(self) -> None:
             ejscreen_csv, dtype={"ID": "string"}, low_memory=False
         )
         self.ejscreen_df.rename(
-            columns={"ID": self.GEOID_FIELD_NAME}, inplace=True
+            columns={
+                "ID": self.GEOID_FIELD_NAME,
```

Comment:
Nice cleanup! 

If you want, we can add a TODO to move this into the EJSCREEN ETL.